### ISSUE TITLE
Discord deckboard does not update

### ISSUE THREAD

**Issue by longopy** [aislandener/discord-deckboard#18]
Deckboard continues to indicate that there is an update. No matter how hard I try to update, it won't update to the latest version. 
![imagen](https://github.com/aislandener/discord-deckboard/assets/17580010/4220d3c2-a392-4f39-b07c-b2699458d14e)
![imagen](https://github.com/aislandener/discord-deckboard/assets/17580010/5829b7bf-8e3b-440a-8472-bcbc5dc07dd8)
I have also tried downloading the package manually and overwriting it, but it still says it is version 1.1.2 and that I need to update.

**Comment by alexonpeace** [comment]
Yeah, this is annoying

**Comment by Thornbrier** [comment]
This is still an issue. Not only in the Deckboard itself, but even downloading from here, it still displays as being 1.1.2 saying I need to update, regardless of how many times I try. The functions are not working either. All I've tried so far is mute/unmuting myself, but it doesn't register at all.

**Comment by Tayra112** [comment]
Yep same thing here, I tried to compile it myself to see if it would change anything, but no.

**Comment by joshrosario310** [comment]
Same issue, even manual installation of pre-compiled. I am not familiar enough with JavaScript to attempt to compile myself but I did notice that between version 1.1.0 and 1.1.2, extension.yml changed from
`repository: "https://github.com/aislandener/discord-deckboard"`
to
`repository: "https://github.com/osmancitci/discord-deckboard"`
for a version 1.1.1 which isn't in this repo but is located at osmancitci

I wonder if because 1.1.2 doesn't exist at osmancitci/discord-deckboard if this line needs to be changed back to aislandener/discord-deckboard for the Deckboard application to actually find the update?

**Comment by awgui** [comment]
any news?

**Comment by joshrosario310** [comment]
@FernandoFuvio when I edit that binary file, I find four instances of 1.1.2 but when I change them and restart Deckboard, it shows as uninstalled in the extensions.

**Comment by FernandoFuvio** [comment]
Did you use Notepad++ to replace the text or another application like Windows Notepad?

**Comment by joshrosario310** [comment]
I used VSCode Find & Replace

**Comment by FernandoFuvio** [comment]
> I used VSCode Find & Replace

Maybe it changed coding format or something. Try replacing using Notepad++. Here worked fine.

**Comment by longopy** [comment]
> You can edit the 'asar' file in Notepad++ and replace everywhere that is "1.1.2" to "1.1.3". Restart Deckboard and it stops asking to update. The 'asar' file is in C:\Users\youruser\deckboard\extensions

This does not notify you of the update, but you are still using internally version 1.1.2.

**Comment by FernandoFuvio** [comment]
> > You can edit the 'asar' file in Notepad++ and replace everywhere that is "1.1.2" to "1.1.3". Restart Deckboard and it stops asking to update. The 'asar' file is in C:\Users\youruser\deckboard\extensions
> 
> This does not notify you of the update, but you are still using internally version 1.1.2.

No, the issue is that the developer forgot to update the internal info of the version in v1.1.3. 

You are in version 1.1.3, but it show in Deckboard that you're in 1.1.2 because that's what it says in 1.1.3 asar.

**Comment by longopy** [comment]
Well, I don't see the changes in version 1.1.3 reflected in that packaging. Toggle Microphone option with icon change

**Comment by FernandoFuvio** [comment]
> Well, I don't see the changes in version 1.1.3 reflected in that packaging. Toggle Microphone option with icon change

Hmmm.

Here you can see the changes. Maybe we can implement on the asar:
https://github.com/aislandener/discord-deckboard/compare/v1.1.2...v1.1.3

**Comment by longopy** [comment]
> > Well, I don't see the changes in version 1.1.3 reflected in that packaging. Toggle Microphone option with icon change
> 
> Hmmm.
> 
> Here you can see the changes. Maybe we can implement on the asar: [v1.1.2...v1.1.3](https://github.com/aislandener/discord-deckboard/compare/v1.1.2...v1.1.3)
I tried to compile the main branch version and when I try to add the .asar file the deckboard program does not detect it.

**Comment by longopy** [comment]
> > Well, I don't see the changes in version 1.1.3 reflected in that packaging. Toggle Microphone option with icon change
> 
> Hmmm.
> 
> Here you can see the changes. Maybe we can implement on the asar: [v1.1.2...v1.1.3](https://github.com/aislandener/discord-deckboard/compare/v1.1.2...v1.1.3)

I tried to compile the main branch version and when I try to add the .asar file the deckboard program does not detect it.

**Comment by Simego** [comment]
have you tried removing the extension and installing directly from store? the version values come from this: https://github.com/rivafarabi/deckboard-extensions/blob/master/extensions/discord-deckboard.yml and the version seems alright
